### ISSUE TITLE
Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'

### ISSUE THREAD

**Issue by joshuabowers** [jkettmann/graphql-passport#4]
Upgrading from 0.1.5 to 0.2.0 causes the above error to be generated when running in a server environment which does not use babel.

Error dump:
```
Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
Require stack:
- /project/node_modules/graphql-passport/lib/index.js
- /project/server/authentication.js
- /project/index.js
    at Object.<anonymous> (/project/node_modules/graphql-passport/lib/index.js:3:30)
    at Generator.next (<anonymous>)
    at Object.<anonymous> (/project/server/authentication.js:1)
    at Generator.next (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/project/node_modules/graphql-passport/lib/index.js',
    '/project/server/authentication.js',
    '/project/index.js'
  ]
}
```

That is, from where my server code is attempting to set up passport auth using a GraphQLLocalStrategy object, the node module lib file fails. Looking at the source for index.js:

```js
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "buildContext", {
  enumerable: true,
  get: function get() {
    return _buildContext["default"];
  }
});
Object.defineProperty(exports, "GraphQLLocalStrategy", {
  enumerable: true,
  get: function get() {
    return _GraphQLLocalStrategy["default"];
  }
});

var _buildContext = _interopRequireDefault(require("./buildContext"));

var _GraphQLLocalStrategy = _interopRequireDefault(require("./GraphQLLocalStrategy"));
```

Clearly, this is depending on babel for doing things.

My server environment isn't currently using babel, and I'd rather not go to the trouble of setting up babel just for this strategy. Downgrading 0.1.5 seems to work as expected for now, but I'd like to ensure I have bug fixes and new features as they become available.

Is babel, going forward, a dependency? If so, the readme and package.json should probably be updated to reflect that.

(I cannot, of course, rule out that _I'm_ doing something weird on my end which caused this, but, again, no babel in the server environment. About the only thing I can think of which might be causing this would be esm.) 

Method to reproduce:

In a fresh project:

```
mkdir test && cd test
yarn init --yes
yarn add graphql-passport
```

No other dependencies, the generated `node_modules/graphql-passport/lib/index.js` file has the babel dependency.

**Comment by jkettmann** [comment]
Sorry for the late reply and thanks for this issue. Babel shouldn't be a requirement. I'll have a look into this.

**Comment by jkettmann** [comment]
I released a new version 0.2.1. I hope this fixes your problems
